homeassistant/components/amcrest/__init__.py

```diff
@@ -103,6 +104,7 @@ def _has_unique_names(devices):
         vol.Optional(CONF_SENSORS): vol.All(
             cv.ensure_list, [vol.In(SENSORS)], vol.Unique()
         ),
+        vol.Optional(CONF_EVENT): cv.ensure_list,
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
added support for config flow